Content.Shared/_RMC14/Atmos/SharedRMCFlammableSystem.cs

```diff
@@ -129,10 +147,9 @@ private void OnTileFireMapInit(Entity<TileFireComponent> ent, ref MapInitEvent a
 
     private void OnTileFireVaporHit(Entity<TileFireComponent> ent, ref VaporHitEvent args)
     {
-        if (_net.IsClient)
-            return;
-
+        if (_net.IsClient) return;
```

Comment:
You need to get your editor to parse the .editorconfig at the project's root, these syntax changes are unnecessary and also against the style guide that's in the repo